requirements.txt

```diff
@@ -15,3 +15,4 @@ omegaconf==2.0.1rc4
 lmdb==0.98
 termcolor==1.1.0
 datasets==1.2.1
+pycocotools==2.0.2
```

Comment:
We will need to import this to fbcode and check how to add this dependency to the TARGETS file.

Reply:
(added `pycocotools` to TARGETS internally)